project_task_additional_attributes/models/project_task.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2023 Quartile Limited
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+
+from odoo import _, api, fields, models
+from odoo.exceptions import ValidationError
+
+
+class ProjectTask(models.Model):
+    _inherit = "project.task"
+
+    account_manager = fields.Many2one("res.users")
+    view_id = fields.Char(string="View ID", help="List of numeric values")
+    feasible = fields.Selection([("yes", "Yes"), ("no", "No")], string="Feasible?")
+
+    @api.constrains("view_id")
+    def _check_view_id_numeric(self):
+        for task in self:
+            if task.view_id and not task.view_id.isdigit():
+                raise ValidationError(_("View ID must contain only numeric values."))
```

Comment:
@rinaldifirdaus I guess this doesn't go with client's request.  Please follow up and check.